app/src/main/res/values/strings.xml

```diff
@@ -326,5 +329,9 @@
     <string name="title_activity_chapter_downloads">ChapterDownloadsActivity</string>
     <string name="delete_chapters">Delete Chapters</string>
 
+    <!-- v0.8 Strings -->
+
+    <string name="immersive_mode" translatable="false">Show Full Screen</string>
+    <string name="immersive_mode_description" translatable="false">Hide the system bars when reading</string>
 
```

Comment:
Should not be "translatable=false" as these string will be needing translations. For now add them in all the other string files too.